Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -89,15 +89,15 @@ extension HTTPClient {
     /// Represent HTTP request.
     public struct Request {
         /// Request HTTP version, defaults to `HTTP/1.1`.
-        public var version: HTTPVersion
```

Comment:
Johannes suggest to drop version from request completely (see discussion in #56)

Reply:
even better! done in https://github.com/swift-server/async-http-client/pull/67/commits/73c400f69ecefedc10ed22bf49a09eefe21dc832